apps/api/messaging/src/modules/email/email.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Controller, Get } from '@nestjs/common';
+import { ApiBearerAuth, ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+
+import { EmailService } from './email.service'
+
+@ApiBearerAuth()
+@ApiTags('email')
+@Controller("email")
+export class EmailController {
+  constructor(private readonly emailService: EmailService) {}
+
+  @Get()
+  @ApiOperation({summary: 'test'})
+  @ApiResponse({status: 400, description: 'Bad Request'})
```

Comment:
Good use of swagger, also you can stack responses 
![image](https://user-images.githubusercontent.com/54650861/190862832-159f7648-21a9-4f89-8d87-c0f863da186a.png)